app/routes/reports.tsx

```diff
@@ -24,10 +40,17 @@ interface IPageData {
 export const loader: LoaderFunction = async () => {
 	try {
 		const reports = await fetchReports();
+		const reportIds = new Set();
+		// If it a report cmsId is not unique, it will be filtered out, this seems to always return the first report
```

Comment:
That's odd. I would think that the reportId in the CMS would be unique.

Reply:
remember we have a double mint problem rn, so some reports are in the results twice

Reply:
yeah i copied over @thebeyondr's fix from #39